src/gui/dialogs/liveadjust_z.hpp

```diff
@@ -8,6 +8,23 @@
 #include "window_arrows.hpp"
 #include "../../lang/i18n.h"
 
+/* class WindowScale : public window_aligned_t { */
+class WindowScale : public AddSuperWindow<window_frame_t> {
+    window_numb_t scaleNum0;
+    window_numb_t scaleNum1;
+    window_numb_t scaleNum2;
+    point_i16_t point;
```

Comment:
comment `point` idk what it does ...

Reply:
is it origin (top left)?

Reply:
U do not need `point `at all, U stored it into parent `rect` in ctor
So just use `rect.TopLeft() `